subworkflows/local/prepare_references.nf

```diff
@@ -65,6 +67,9 @@ workflow PREPARE_REFERENCES {
         BEDTOINTERVALLIST( GET_RRNA_TRANSCRIPTS.out.bed.map {it -> [ [id:it.name], it ]}, ch_dict )
         ch_versions = ch_versions.mix(BEDTOINTERVALLIST.out.versions)
 
+        UNTAR_VEP_CACHE (ch_vep_cache)
+        ch_versions = ch_versions.mix(UNTAR_VEP_CACHE.out.versions)
```

Comment:
What happens when you send in a vep cache in the for of a directory and not a tarball?

Reply:
You are right! I fixed it in the config